src/minimum_dependencies/_core.py

```diff
@@ -72,11 +72,10 @@ def minimum_version(requirement: Requirement, fail: Fail = Fail.FALSE) -> Versio
     The minimum version available on PyPi for the given requirement.
     """
     if not requirement.specifier:
-        msg = f"No version specifier for {requirement.name} in install_requires."
+        msg = f'Could not parse a version specifier from "{requirement}"'
         if fail:
             raise ValueError(msg)
 
-        msg += "\nUsing lowest available version on PyPi."
```

Comment:
I don't like removing this message please add it back, it informs the user of exactly what kind of choice is being made due to the warning.

Line 94 will not be reached in this case and so the message must be added here.